src/EnergyPlus/DXCoils.cc

```diff
@@ -2411,13 +2411,17 @@ void GetDXCoils(EnergyPlusData &state)
                 ErrorsFound = true;
             }
         }
-        thisDXCoil.RatedTotCap(1) = Numbers(1);
-        thisDXCoil.RatedSHR(1) = Numbers(2);
-        thisDXCoil.RatedCOP(1) = Numbers(3);
-        thisDXCoil.RatedAirVolFlowRate(1) = Numbers(4);
-        if (!lNumericBlanks(5)) {
-            thisDXCoil.InternalStaticPressureDrop = Numbers(5);
-            thisDXCoil.RateWithInternalStaticAndFanObject = true;
+        state.dataDXCoils->DXCoil(DXCoilNum).RatedTotCap(1) = Numbers(1);
```

Comment:
?? Why did you take out the reference variable?  We are working hard to clean up these array usages and replace them with more readable references, and this undoes a bunch of them for some reason.

Reply:
This reference variable(**thisCoil**) was made after we created our branch and I think the auto merge has picked the fully qualified names at places where we've some modifications (from our branch). I've replaced them now. Thank-you for pointing it out.